src/Data/IPPEmployee.php

```diff
@@ -170,6 +170,16 @@ public function __construct($keyValInitializers=array(), $verbose=FALSE)
 	 * @var com\intuit\schema\finance\v3\IPPIntuitAnyType
 	 */
 	public $EmployeeEx;
+	/**
+	 * @Definition Hourly cost rate of the Employee. QBO only. QBD Unsupporetd field.
```

Comment:
Typo: "Unsupporetd".

```suggestion
	 * @Definition Hourly cost rate of the Employee. QBO only. QBD Unsupported field.
```